cookbooks/bcpc-hadoop/recipes/smoke_test_wrapper.rb

```diff
@@ -80,6 +80,17 @@
   user test_user
 end
 
+hive_jdbc_url = 'jdbc:hive2://'
+zookeeperQuorum = node[:bcpc][:hadoop][:zookeeper][:servers].map { |s|
```

Comment:
Please check with rubocop. Particularly multiline `{}` blocks are discouraged for `do` blocks instead.